worlds/celeste64/Options.py

```diff
@@ -1,25 +1,107 @@
 from dataclasses import dataclass
 
-from Options import Range, DeathLink, PerGameCommonOptions
+from Options import Choice, Range, Toggle, DeathLink, PerGameCommonOptions
 
 
-class StrawberriesRequired(Range):
-    """How many Strawberries you must receive to finish"""
-    display_name = "Strawberries Required"
-    range_start = 0
-    range_end = 20
-    default = 15
-
 class DeathLinkAmnesty(Range):
     """How many deaths it takes to send a DeathLink"""
     display_name = "Death Link Amnesty"
     range_start = 1
```

Comment:
Perhaps this is out of scope for this change, but I think it's worth pointing out that the other two worlds with DeathLink Amnesty both have it as a range starting at 0. Unsure if you'd want to change it, but I figured you should know

Reply:
"Amnesty" usually means being forgiven/pardoned. So setting your "forgiveness" to 1 and that being the _minimum_ doesn't really make sense. Looking at how the other games actually word the option though, I would say they don't make sense either. I think "Amnesty" may just not be a good word for what's trying to be conveyed here

Reply:
And given that the word is "amnesty" one would assume the minimum amount of forgiveness you can have is none, which maps much better to 0 than to 1

Reply:
You can mark this as resolved then